TrigScint/src/TrigScint/SimQIE.cxx

```diff
@@ -62,7 +61,8 @@ float SimQIE::ADC2Q(int ADC) {
   for (int i = 1; i < 4; i++) {  // to get the subrange
     if (v1 > nbins_[i]) ss++;
   }
-  int cc = 64 * rr + nbins_[ss];
+  // cc is unused, should it be? FIXME
+  // int cc = 64 * rr + nbins_[ss];
```

Comment:
@bryngemark   can I delete this?
```suggestion
```

Reply:
i don't know about this, it's @GNiramay's code, and he may or may not be around to answer :)